File: ServerApp/Config/Objects/Items/Car/Parts/Wheels.cs
using System;

namespace LBTDTools.ServerApp.Config.Objects.Items.Car.Parts
{
    public class Wheels
    {
        public string Name  { get; set; }
        public string WheelWidth { get; set; }
        public string ProfileSize { get; set; }
        public string Radius { get; set; }

        public string GetFullName()
        {
            return Name + " " + WheelWidth + "/" + ProfileSize + "/R" + Radius;
        }
    }
}

File: ClientApp/Config/Objects/Items/Ticket/Car/Parts/Wheels.ts
export default class Wheels
{
    public name: string;
    public wheelWidth: string;
    public profileSize: string;
    public radius: string;

    constructor(name: string, wheelWidth: string, profileSize: string, radius: string)
    {
        if (name == "")
        {
            this.name = " без изменений";
            this.wheelWidth = "";
            this.profileSize = "";
            this.radius = "";
        }
        else
        {
            this.name = name;
            this.wheelWidth = wheelWidth;
            this.profileSize = profileSize;
            this.radius = radius;
        }
    }
}